src/objc/imap/MCOIMAPIdentity.mm

```diff
@@ -33,6 +33,11 @@ - (id) initWithMCIdentity:(mailcore::IMAPIdentity *)identity
     return self;
 }
 
+- (id) init
+{
+    return [self initWithMCIdentity:new nativeType()];
```

Comment:
the allocated nativeType() needs to be released.
Take a look at MCOIMAPFolder for reference.